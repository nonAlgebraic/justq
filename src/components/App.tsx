import { h, FunctionComponent } from 'preact';
import * as styles from './App.styles.js';

const App: FunctionComponent = () => (
  <div className={styles.root}>Hello, world!</div>
);

export default App;
