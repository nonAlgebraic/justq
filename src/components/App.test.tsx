import { h } from 'preact';
import { render } from '@testing-library/preact';
import { axe } from 'jest-axe';
import App from './App';

describe('App', () => {
  it('Renders', () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });

  it('Has no a11y violations', async () => {
    const { container } = render(<App />);

    expect(await axe(container)).toHaveNoViolations();
  });
});
