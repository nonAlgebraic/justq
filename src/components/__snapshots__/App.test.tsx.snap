// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Renders 1`] = `
.emotion-0 {
  color: red;
}

<div>
  <div
    class="emotion-0"
  >
    Hello, world!!!
  </div>
</div>
`;
